content/en/docs/reference/kubectl/cheatsheet.md

```diff
@@ -151,8 +151,8 @@ kubectl get deployment my-dep                 # List a particular deployment
 kubectl get pods                              # List all pods in the namespace
 kubectl get pod my-pod -o yaml                # Get a pod's YAML
 
-# Describe commands with verbose output
-kubectl describe nodes my-node
```

Comment:
Generally, all nodes are not described, so the output content is too much, it's not like `get pods` shows a list.

Shown here are the most commonly used commands, right?